From e2a0edd89ffd2b4b6824a9e018630446779e95dc Mon Sep 17 00:00:00 2001
From: Tuukka Toivonen <tuukka.toivonen@intel.com>
Date: Mon, 13 May 2013 15:58:02 +0300
Subject: [PORT FROM MAIN] bionic: Update atomisp.h

BZ: 105904

Update atomisp.h from the kernel to include the new menu-style
control V4L2_CID_VFPP.

Related patches:
atomisp2: replace V4L2_CID_ENABLE_VFPP with menu-style V4L2_CID_VFPP
(Change-Id: If33928d10559426a192407e197d7045c81fea8eb)
libcamera2: replace V4L2_CID_ENABLE_VFPP with menu-style V4L2_CID_VFPP
(Change-Id: I80ed78f029730213ca115d0cdb672bfa4b3eb438)

Change-Id: I683278eda519b7df5608f5e43bca1c8a59b66c35
Orig-Change-Id: Ie930d8c6045cb066752335fc474bfd062a980655
Signed-off-by: Tuukka Toivonen <tuukka.toivonen@intel.com>
Reviewed-on: http://android.intel.com:8080/107323
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Ailus, Sakari <sakari.ailus@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Kaartinen, Jukka O <jukka.o.kaartinen@intel.com>
Tested-by: Koski, Anttu <anttu.koski@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 libc/kernel/common/linux/atomisp.h |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/libc/kernel/common/linux/atomisp.h b/libc/kernel/common/linux/atomisp.h
index 7745602..a57ea3a 100644
--- a/libc/kernel/common/linux/atomisp.h
+++ b/libc/kernel/common/linux/atomisp.h
@@ -703,6 +703,11 @@ struct v4l2_private_int_data {
 /* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
 #define V4L2_CID_ATOMISP_CONTINUOUS_RAW_BUFFER_SIZE   (V4L2_CID_CAMERA_LASTP1 + 23)
 #define V4L2_CID_ATOMISP_CONTINUOUS_VIEWFINDER   (V4L2_CID_CAMERA_LASTP1 + 24)
+#define V4L2_CID_VFPP (V4L2_CID_CAMERA_LASTP1 + 25)
+#define ATOMISP_VFPP_ENABLE 0
+/* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
+#define ATOMISP_VFPP_DISABLE_SCALER 1
+#define ATOMISP_VFPP_DISABLE_LOWLAT 2
 #define V4L2_BUF_FLAG_BUFFER_INVALID 0x0400
 #define V4L2_BUF_FLAG_BUFFER_VALID 0x0800
 /* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
@@ -713,3 +718,4 @@ enum {
 /* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
 };
 #endif
+
-- 
1.7.4.1

